pkgs/development/web/mailcatcher/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, bundlerEnv, ruby, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "mailcatcher-${version}";
+
+  version = (import ./gemset.nix).mailcatcher.version;
+
+  env = bundlerEnv {
+    name = "${name}-gems";
+
+    inherit ruby;
+
+    gemdir = ./.;
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  phases = ["installPhase"];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    makeWrapper ${env}/bin/mailcatcher $out/bin/mailcatcher
+    makeWrapper ${env}/bin/catchmail $out/bin/catchmail
+  '';
+
+  meta = with lib; {
+    description = "Catches mail and serves it through a dream";
```

Comment:
I assume you got the description from upstream, but I'd rather see something with any meaning. The name already says that it catches mail, so please make the words count in the description.